sickbeard/providers/torrentz.py

```diff
@@ -90,7 +90,7 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
 
                             title_raw = item.title.text
                             # Add "-" after codec and add missing "."
-                            title = re.sub(r'([xh][ .]?264|xvid)', r'\1-', title_raw).replace(' ','.') if title_raw else ''
+                            title = re.sub(r'([xh][ .]?264|xvid) ', r'\1-', title_raw).replace(' ','.') if title_raw else ''
```

Comment:
I think the chance of it matching a newline or a tab in a title is fairly low, however the regex could make the space only match a little more obvious using: `r'([xh][ .]?264|xvid)( )'`
(or `r'([xh][ .]?264|xvid)(?: )'` if you prefer).